tests/com/rcv/TabulatorTests.java

```diff
@@ -128,15 +135,23 @@ public static void setup() {
   // function: testPortlandMayor
   // purpose: test tabulation of Portland contest
   @Test
-  @DisplayName("Portland Mayor 2015")
+  @DisplayName("2015 Portland Mayor")
   void testPortlandMayor() {
-    runTabulationTest("config_2015_portland_mayor.json", "2015_portland_mayor_expected.json");
+    runTabulationTest("2015_portland_mayor");
   }
 
+  // function: test2013MinneapolisMayorScale
+  // purpose: test large scale (1,000,000+) cvr contest
+  @Test
+  @DisplayName("2013 Minneapolis Mayor Scale")
+  void test2013MinneapolisMayorScale() {
```

Comment:
How long does this take? It's good to have, but might get a little annoying if it runs every time we want to run the test suite.